### PR TITLE
Move debugging config page into configuring-ember section

### PR DESCRIPTION
**Pull request by michaelrkn**
Configuration options for debugging belong in the "Configuring Ember" section, not "Understanding Ember".

### CODE REVIEW

File: source/configuring-ember/debugging.md [hunk 1]
```diff
@@ -34,13 +29,6 @@ ENV.APP.LOG_VIEW_LOOKUPS = true;
 Ember.keys(Ember.TEMPLATES)
 ```
 
-#### Handlebars Debugging Helpers
```
Comment (locks) [inline source/configuring-ember/debugging.md]: why did you remove these? Ember Inspector?

Comment (michaelrkn) [inline source/configuring-ember/debugging.md]: They are covered in http://guides.emberjs.com/v1.13.0/templates/development-helpers/. I've tried to keep this page to be about configuration options as much as possible. (It could still use some cleanup, but I wanted to get it in the right place at least.)

**Comment by michaelrkn** [comment]
@locks any other concerns here?

**Comment by locks** [comment]
:shipit:
